tests/test_java.py

```diff
@@ -691,3 +691,19 @@ def test_meminfo_logged(
     ) as profiler:
         snapshot_pid_profile(profiler, application_pid)
         assert "async-profiler memory usage (in bytes)" in caplog.text
+
+
+@pytest.mark.parametrize("in_container", [True])
+def test_java_frames_include_no_semicolons(
```

Comment:
Semicolons should be used between frames, but not as part of a frame. Let's also test here that they are used between frames (so check that a function appears after another function with `;` in between?)

Reply:
Will do.